integrations/influxdb_metrics_dashboard/influxdb_metrics_dashboard.go

```diff
@@ -0,0 +1,389 @@
+}
+
+stop() {
+    echo "Stopping Telegraf..."
+    pkill -f telegraf
+}
+
+restart() {
+    stop
+    start
+}
+
+status() {
+    ps aux | grep telegraf | grep -v grep
+}
+
+case "\$1" in
+  start)
+      start
+      ;;
+  stop)
+      stop
+      ;;
+  restart)
+      restart
+      ;;
+  status)
+      status
+      ;;
+  *)
+      echo "Usage: \$0 {start|stop|restart|status}"
+      exit 1
+      ;;
+esac
+EOT
+
+# Make telegraf executable
+sudo chmod +x /etc/init.d/telegraf
+
+# Setup telegraf to run on boot
+sudo chkconfig --add telegraf
+sudo chkconfig telegraf on
+
+# Start Telegraf Service
+service telegraf start
+`, *stackProps.Env.Region, *stackProps.Env.Region, databaseName, influxDBEndpoint, influxDBUsername, influxDBPassword)
+
+	vpc := awsec2.Vpc_FromLookup(stack, jsii.String("testingInfluxDBMetricsDashboard"), &awsec2.VpcLookupOptions{
+		IsDefault: jsii.Bool(true),
+		Region:    jsii.String(*stackProps.Env.Region),
+		VpcId:     jsii.String(vpcId),
+	})
+	ec2SecurityGroup := awsec2.NewSecurityGroup(stack, jsii.String("TelegrafEC2SG"), &awsec2.SecurityGroupProps{
+		Vpc:               vpc,
+		SecurityGroupName: jsii.String("InstanceSecurityGroup"),
+		Description:       jsii.String("Allow SSH and all outbound traffic"),
+	})
+	ec2SecurityGroup.AddIngressRule(
+		awsec2.Peer_Ipv4(jsii.String("0.0.0.0/0")),
+		awsec2.Port_Tcp(jsii.Number(22)),
+		jsii.String("Allow SSH access"),
+		jsii.Bool(false),
+	)
+	ec2SecurityGroup.AddIngressRule(
+		awsec2.Peer_Ipv4(jsii.String("0.0.0.0/0")),
+		awsec2.Port_Tcp(jsii.Number(8086)),
+		jsii.String("Allow InfluxDB metric port"),
+		jsii.Bool(false),
+	)
+
+	awsec2.NewInstance(stack, jsii.String("TelegrafInfluxDBMetricScraper"), &awsec2.InstanceProps{
+		InstanceType:  awsec2.InstanceType_Of(awsec2.InstanceClass_BURSTABLE2, awsec2.InstanceSize_NANO),
+		MachineImage:  awsec2.NewAmazonLinuxImage(&awsec2.AmazonLinuxImageProps{}),
+		Vpc:           vpc,
+		UserData:      awsec2.UserData_Custom(jsii.String(userDataScript)),
+		Role:          instanceRole,
+		SecurityGroup: ec2SecurityGroup,
+	})
+
+	return stack, nil
+}
+
+func addGrafanaWorkspaceToStack(stack awscdk.Stack, stackProps awscdk.StackProps, databaseName string, grafanaWorkspaceName string) (awscdk.Stack, error) {
+	workspacePolicy := awsiam.NewPolicy(stack, jsii.String("influxdb-dashboard-workspace-policy"), &awsiam.PolicyProps{
+		Statements: &[]awsiam.PolicyStatement{
+			awsiam.NewPolicyStatement(&awsiam.PolicyStatementProps{
+				Actions: &[]*string{
+					jsii.String("timestream:DescribeEndpoints"),
+				},
+				Resources: &[]*string{
+					jsii.String("*"),
+				},
+			}),
+			awsiam.NewPolicyStatement(&awsiam.PolicyStatementProps{
+				Actions: &[]*string{
+					jsii.String("timestream:Select"),
+				},
+				Resources: &[]*string{
+					jsii.String(fmt.Sprintf("arn:aws:timestream:%s:%s:database/%s/table/storage_bucket_series_num", *stackProps.Env.Region, *stackProps.Env.Account, databaseName)),
```

Comment:
Can we give the workspace access to all data in the table?
```suggestion
					jsii.String(fmt.Sprintf("arn:aws:timestream:%s:%s:database/%s/table/*", *stackProps.Env.Region, *stackProps.Env.Account, databaseName)),
```

Reply:
Yeah this would be cleaner considering how many tables there are. I will update to use all tables.